File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.go
// comment.line.double-slash
/* comment.block */

/*
comment.documentation
*/

const PI = 3.14 // constant.numeric
var esc = '\n'  // constant.character.escape
var flag = true // constant.language

type MyType struct {        // entity.name.type
    Field int
}

func myFunction(param int) string { // entity.name.function, variable.parameter
    color := "#ff0000"              // constant.other, string.quoted.double
    if flag {                       // keyword.control
        return color                // keyword.control
    }
    return ""
}

File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.rs
// comment.line.double-slash
/* comment.block */
/// comment.documentation

const PI: f64 = 3.14;          // constant.numeric
let esc = '\n';                // constant.character.escape
let flag = true;               // constant.language

struct MyStruct;               // entity.name.type
fn my_function(param: i32) {   // entity.name.function, variable.parameter
    let color = "#ff0000";     // constant.other, string.quoted.double
    if flag {                  // keyword.control
        println!("{}", color); // support.function
    }
}
